packages/stock-location-next/src/services/__tests__/noop.ts

```diff
@@ -0,0 +1,5 @@
+describe("noop", function () {
```

Comment:
todo: rm file

Reply:
can't remove this as the unit tests will break and the pipeline won't pass, we have the same test in the other modules to make unit-tests pass (fulfillment-module included)

Reply:
you can probably just add that `--passWithNoTests` in your command in the package.json, I think it is better than an empty test no?